src/KafkaFlow.Retry/Durable/Polling/Jobs/RetryDurablePollingJob.cs

```diff
@@ -58,11 +59,12 @@ public async Task Execute(IJobExecutionContext context)
             Guard.Argument(messageHeadersAdapter).NotNull();
             Guard.Argument(messageAdapter).NotNull();
             Guard.Argument(utf8Encoder).NotNull();
+            Guard.Argument(schedulerId).NotNull().NotEmpty();
```

Comment:
Now I understand that you did `CleanupPollingJob` based on this one. Perhaps we can do the same here?

Reply:
Yes the new cleanup job was based on the existent one.